src/strategies/latency.js

```diff
@@ -173,17 +173,17 @@ class LatencyStrategy extends BaseStrategy {
 
 		if (info.nodeList.length == 0) {
 			// only remove the host if the last node disconnected
-			this.broker.localBus.emit("$node.latencySlave.removeHost", node.hostname);
+			this.broker.localBus.emit("$node.latencyReplica.removeHost", node.hostname);
```

Comment:
from what I could tell this is an internal identifier, not part of the public API. but I might be wrong?

Reply:
I believe it's meant for internal use, but devs can subscribe to it if they want to.